robohive/envs/env_base.py

```diff
@@ -263,8 +265,23 @@ def step(self, a, **kwargs):
                                         render_cbk=self.mj_render if self.mujoco_render_frames else None)
         return self.forward(**kwargs)
 
+    @implement_for("gym", None, "0.24")
```

Comment:
I appreciate the effort putting into implement_for and I think its a cool idea.
However, I am curious that wound't it be more cleaner just 
* make a gym_wrapper class that is aligned with new gynasium or old on (pick one
* the wrapper handles adaptation to the other class.

rather than having these changes scattered in various part of robohive.

I am also curious if user have both gym/gynasium installed, what will happen here.

Reply:
1. This recommendation comes from @vmoens. This is also the solution in TorchRL

2. It will prioritize gymnasium over gym: https://github.com/vikashplus/robohive/blob/dev/robohive/utils/__init__.py